ios/browser/api/ads/brave_ads.mm

```diff
@@ -430,11 +413,11 @@ - (void)setAutoDetectedSubdivisionTargetingCode:
 }
 
 - (void)savePref:(NSString*)name {
+  [self savePrefs];
```

Comment:
@kylehickinson this fixes an issue where the listener may not have the most up to date value